apps/src/templates/projects/StudentFilterDropdown.jsx

```diff
@@ -0,0 +1,64 @@
+import React, {Component, PropTypes} from 'react';
+import Radium from 'radium';
+import color from "../../util/color";
+import commonMsg from '@cdo/locale';
+
+const ALL_STUDENTS = '_all_students';
+
+const styles = {
+  filterWrapper: {
+    display: 'inline-flex',
+    alignItems: 'center',
+    fontSize: 14
+  },
+  filterSelect: {
+    margin: 0,
+    color: 'dimgray'
+  },
+  filterText: {
+    color: color.white,
+  },
+};
+
+class StudentFilterDropdown extends Component {
+  constructor(props) {
+    super(props);
+  }
```

Comment:
This can be omitted; [it's the default constructor](http://www.2ality.com/2015/02/es6-classes-final.html#default-constructors-for-classes).